barretenberg/cpp/src/barretenberg/eccvm/eccvm_verifier.cpp

```diff
@@ -101,15 +106,16 @@ bool ECCVMVerifier::verify_proof(const HonkProof& proof)
         batching_scalar *= ipa_batching_challenge;
     }
 
-    std::array<OpeningClaim<Curve>, 2> opening_claims = { multivariate_to_univariate_opening_claim,
-                                                          { { evaluation_challenge_x, batched_transcript_eval },
-                                                            batched_commitment } };
+    std::array<OpeningClaim, 2> opening_claims = { multivariate_to_univariate_opening_claim,
+                                                   { { evaluation_challenge_x, batched_transcript_eval },
+                                                     batched_commitment } };
 
     // Construct and verify the combined opening claim
-    auto batched_opening_claim =
+    const OpeningClaim batch_opening_claim =
```

Comment:
I think the naming is a bit confusing, it would be great to clean them up. Maybe like sumcheck_batch_opening claims instead of batch_opening_claims and create an exlpiicit opening claim translation_batch_opening_claim, so that we see what the final batch opening claim is. + constify opening_claims